examples/mwaa/environment.yaml

```
@@ -0,0 +1,256 @@
+apiVersion: mwaa.aws.upbound.io/v1beta1
+kind: Environment
+metadata:
+  annotations:
```

Since it took nearly 2 hours to create the resource, we can add the following annotation.
```suggestion
  annotations:
    uptest.upbound.io/timeout: "7200"
```